app/templates/components/ui-table/cell/cell-local-date.hbs

```diff
@@ -0,0 +1,14 @@
+{{#if record.startsAt}}
+  <div>
+    {{moment-format record.startsAt 'MMMM DD, YYYY - hh:mm A'}}
```

Comment:
This is not the correct way. If you're adding the format here and calling the model variable again here what is the point of writing the propertyName and dateFormat in the controller?
Use something like (get record column.propertyName) in the template.
Use the file app/templates/components/ui-table/cell/cell-simple-date.hbs as reference.